docs/operating-scylla/rest.rst

```diff
@@ -32,7 +32,7 @@ The following example shows using the Swagger UI with Docker.
 
     .. code:: sh
 
-      docker run --name some-scylla -p 10000:10000 -d scylladb/scylla:5.0.4 --api-address 0.0.0.0
+      docker run --name some-scylla -p 10000:10000 -d scylladb/scylla:latest --api-address 0.0.0.0
```

Comment:
currently, the title of this commit reads:

> In example to view Swagger UI for REST API, use latest docker tag 

would be better to add a prefix to indicate the area/subdirectory where this change touches like:

> docs: rest.rst: use latest docker tag to view Swagger UI for REST API

Reply:
Hi @tchaikov ,

Thanks! I swapped out my commit message for yours.